flowtools/peacoqc/PeacoQC_implementation.R

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env Rscript
+
+if (!requireNamespace("BiocManager", quietly = TRUE))
+  install.packages("BiocManager")
+
+BiocManager::install("PeacoQC")
+
```

Comment:
```suggestion
```

This is not needed because the script will run in an environment where the package is available.